lib/extract-dependency.js

```diff
@@ -5,7 +5,7 @@ module.exports = function extractDependency (pkg, branch) {
   const config = pkg.greenkeeper || {}
   const branchPrefix = config.branchPrefix || 'greenkeeper/'
 
-  if (!branch.startsWith(branchPrefix)) throw new Error('Not a Greenkeeper pull request.')
+  if (!branch.startsWith(branchPrefix)) throw new Error('Not a Greenkeeper branch.')
```

Comment:
I can change all these back, but I found it confusing (both as a user and dev) that it wasn't running on pull request builds, but then says in other messages that it has to run in a pull request build.